pombola/south_africa/static/images/navbar/pa_icon.svg

```diff
@@ -0,0 +1,46 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0"
```

Comment:
This is a logo, right? 
Also it's not an icon.
Therefore I'd recommend calling it pa_logo.svg instead.